backends/p4tools/modules/testgen/targets/bmv2/test/BMV2ProtobufXfail.cmake

```diff
@@ -9,6 +9,13 @@
 ####################################################################################################
 # These are failures in P4Testgen that need to be fixed.
 
+p4tools_add_xfail_reason(
```

Comment:
These are new errors that arose due to this refactoring?

Reply:
Because you turned ON running the tests below I assume?

Reply:
Yes, exactly. These were errors that existed before but were not captured yet.